util/slice/slice.go

```diff
@@ -1,21 +0,0 @@
-package slice
-
-func RemoveString(slice []string, element string) []string {
-	for i, v := range slice {
-		if element == v {
-			ret := make([]string, 0, len(slice)-1)
-			ret = append(ret, slice[:i]...)
-			return append(ret, slice[i+1:]...)
-		}
-	}
-	return slice
```

Comment:
I assume that you decided that it wasn't worth keeping this file for a simplified version of this helper that uses `DeleteFunc`? i.e. `return slices.DeleteFunc(slice, func (x string) bool { return x == element }`

Reply:
I considered it, but didn't do it as:
* It behaves differently from the original (modifying the input slice in place). This is sort of surprising, as it also returns the resulting slice. The `slices` version documents this for us.
* `RemoveString()` only operated on arrays of `string`s, not generic arrays, so overall encouraging it to stay around and potentially get duplicated to add `RemoveMyOtherType()` was worse long term. Possibly genericing it would have been useful, but probably worse as a long term option.

Reply:
Yea I considered making it a generic `RemoveElement` as well. 

Thanks for noting this; I don't have a strong opinion on this but wanted to make sure the rationale was documented since the decision seemed intentional